app/helpers.php

```diff
@@ -176,3 +176,7 @@ function minutesToHours($minutes)
 
     return sprintf('%02.2d:%02.2d', floor($seconds / 3600), $seconds / 60 % 60);
 }
+
+function is_local_environment() {
+    return env("APP_IS_LOCAL") == "true" || App::environment('staging');
```

Comment:
A staging env isn't really local, we should probably restrict this to just return the env true/false